cases/0011/0011.py

```diff
@@ -0,0 +1,69 @@
+from petabtests import *
+from petab.C import *
+import petab
+
+import pandas as pd
+
+
+test_id = 11
+
+# problem --------------------------------------------------------------------
+
+model = DEFAULT_MODEL_FILE
+
+condition_df = pd.DataFrame(data={
+    CONDITION_ID: ['c0'],
+    'a0': [3],
```

Comment:
For this case, we would write the species ID here, not the target of the initial assignment, not?
Maybe the initial assignment should be removed from the model here. Species IDs in the condition table were
meant to avoid using initial assignments. Having both may be unnecessarily complex.

Reply:
adjusted. Want to have another look? I get an error in amici regarding a mismatch of parameter and scale mappings. May be the model, or some bug. Currently, 0006, 0010 and 0012 fail in amici.